src/test/java/com/limechain/sync/fullsync/BlockExecutorTest.java

```diff
@@ -23,68 +25,46 @@
 import io.emeraldpay.polkaj.types.Hash256;
 import lombok.extern.java.Log;
 import org.apache.commons.lang3.ArrayUtils;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.junit.jupiter.MockitoExtension;
 
 import java.io.IOException;
 import java.math.BigInteger;
 
 import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+//@formatter:off
 @Log
-@ExtendWith(MockitoExtension.class)
-@Disabled("This test is currently broken in CI")
```

Comment:
@David-Petrov ^

Reply:
I see this as a major blocker. What this means for now is that GH will reject any kind of tests we'd like to add around runtime invocations; which was one of the main goals of this PR.

Reply:
@Zurcusa you could link to this conversation and resolve it, when (and if) your PR resolves this issue. Thanks in advance 👍